explorer/src/components/layout/SectionHeader.tsx

```diff
@@ -59,6 +60,19 @@ export const SectionHeader: FC = () => {
     }
   }, [])
 
+  const getNavbarTitle = useCallback(
+    (item: Route, isActive: boolean) => {
+      if (isLargeDesktop) {
+        return item.title
+      }
+      if (isDesktop) {
+        return item.title.split(' ')[0]
```

Comment:
This is a bit too simplistic. For example "Permanent Storage" becomes "Permanent" which really means nothing. Could we store item titles in a way that they have a `full` and `shortened` version?

Reply:
@iamnamananand996 any thoughts on this?